german/suchen/suchen.md

```diff
@@ -0,0 +1,6 @@
+Ich suche im Boot herum und entdecke einen alten Ordenr, den ich als Paddel benutzen kann.
+
+Was mache ich jetzt?
+
+[Ich fange an damit zu paddeln.] (../paddeln/paddeln.md)
+[Ich werfe den Ordner ueberbord, denn es ist zuviel Arbeit zu paddeln] (../werfen/werfen.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.